### ISSUE TITLE
Let the Meteor CLI use custom cert files

### ISSUE THREAD

**Issue by kitensei** [meteor/meteor#5318]
***1 Upvote*** I am running an Ubuntu 14.04 LTS behind a corporate SSL proxy, I have installed the certificate in the trusted certificates list on y machine (cURL works seamlessly), and when I use node I have configured the `.npmrc` file with the `ca='/etc/ssl/certs/mycert.pem'` and it works great.

Now I am trying to use meteor and when installing a meteor package I get the following:

```
vagrant@mean:/var/www/html/sms-monitoring$ meteor add numtel:mysql
Unable to update package catalog (are you offline?)

If you are using Meteor behind a proxy, set HTTP_PROXY and HTTPS_PROXY environment variables or see this page for more details:
https://github.com/meteor/meteor/wiki/Using-Meteor-behind-a-proxy

 => Errors while adding packages:

While downloading numtel:mysql@1.0.3...:
error: UNABLE_TO_VERIFY_LEAF_SIGNATURE

Your package catalog may be out of date.
Please connect to the internet and try again.

```

Can someone help me ?

**Comment by avital** [comment]
Hmm. I don't think there's currently a way to tell the meteor tool to point to a cert file when it calls `request` internally. (Meteor doesn't use npm to install packages so your `.npmrc` file won't have an effect)

A workaround for now is to ask get the meteor tool to not verify certificates at all by running `export NODE_TLS_REJECT_UNAUTHORIZED=0` before calling Meteor.

If this is important, we could have a `.meteorrc` file

**Comment by kitensei** [comment]
So it isn't possible today to "install a ssl certificate" for meteor ? The only solution is to use the `NODE_TLS_REJECT_UNAUTHORIZED=0` ?

I was hoping a solution like the npm `cafile=xxx` option

**Comment by avital** [comment]
Unfortunately, that's unavailable at the moment.

**Comment by MasterAM** [comment]
+1 for `.meteorrc`, could be very handy. I was thinking about an app-level runcom file that will be sourced automatically when running `meteor`.

**Comment by abernix** [comment]
CAFILE is supported like `npm`.  This appears to have been resolved with 0aa1291fd65d13e9cefadb1cf94a5d966d827e24 from #4757.
